templates/fine-tune-llm_v2/README.md

```diff
@@ -0,0 +1,119 @@
+# Fine-tuning Llama-2/Mistral models with Anyscale
+
+This template offers starter configurations to fine-tune LLM models.
+
+### Supported base models
+
+- mistralai/Mistral-7B-Instruct-v0.1
+- mistralai/Mixtral-8x7b
+- meta-llama/Llama-2-7b-hf
+- meta-llama/Llama-2-7b-chat-hf
+- meta-llama/Llama-2-13b-hf
+- meta-llama/Llama-2-13b-chat-hf
+- meta-llama/Llama-2-70b-hf
+- meta-llama/Llama-2-70b-chat-hf
+
+# Step 1 - Launch a fine-tuning job
+
+We have provided different example configurations under the `training_configs`
+directory for different base models and instance types. You can use these as a
+starting point for your own fine-tuning jobs.
+
+[Optional] you can get a WandB API key from [WandB](https://wandb.ai/authorize) to track the finetuning process.
+
+Next, you can launch a fine-tuning job where the WandB API key is passed as an environment variable.
+
+```shell
+# Launch a full-param fine-tuning job for Llama 7b with 16 g5.4xlarge instances
+WANDB_API_KEY={YOUR_WANDB_API_KEY} python train.py job_compute_configs/aws.yaml training_configs/full_param/llama-2-7b-512-16xg5_4xlarge.yaml
+
+# Launch a LoRA fine-tuning job for Llama 7b with 16 g5.4xlarge instances
+WANDB_API_KEY={YOUR_WANDB_API_KEY} python train.py job_compute_configs/aws.yaml training_configs/lora/llama-2-7b-512-16xg5_4xlarge.yaml
+```
+
+Once you submit the command, in the terminal you will see the link to your finetuning job. Ex:
+
+```
+View the job in the UI at https://console.anyscale.com/jobs/prodjob_62is21vur3fwl5y5xkc9u1t3ll
```

Comment:
i dont think you can test this on OA yet. You need to make the follow up change of not using jobs to test it on OA